bigbluebutton-html5/imports/ui/components/settings/component.jsx

```diff
@@ -228,7 +228,10 @@ class Settings extends Component {
           description: intl.formatMessage(intlMessages.SaveLabelDesc),
         }}
         dismiss={{
-          callback: () => Settings.setHtmlFontSize(this.state.saved.application.fontSize),
+          callback: () => {
+            Settings.setHtmlFontSize(this.state.saved.application.fontSize);
+            this.props.mountModal(null);
```

Comment:
The mountModal is used in confirm and dismiss callbacks. Can you destruct it and use directly?